aea/helpers/_import_wrapper.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Import patcher."""
+
+import builtins
+
+
+class ImportWrapper:
+    """This class patches the built-in __import__ function."""
+
+    _builtin_import = builtins.__import__
+
+    @staticmethod
+    def wrapped_import(name: str, globals=None, locals=None, fromlist=(), level=0):
+        """
+        The patched __import__ function.
+
+        It acts as a wrapper to the built-in __import__ function.
+
+        It only gets activated if the first part of the import path
+        is equal to "packages". Otherwise, it forwards the call to
+        the builtins.__import__ function.
+
+        For documentation on the parameters,
+        please look at https://docs.python.org/3/library/functions.html#__import__.
+        """
+        kwargs = dict(globals=globals, locals=locals, fromlist=fromlist, level=level)
+        # print(name, globals, locals, fromlist, level)
+        parts = name.split(".")
+        root = parts[0]
+        if root == "packages" and level == 0:
```

Comment:
without `level == 0`, many conflicting packages mentioned above get processed in our handler. However, this is only a workaround, since in some AEA package code users could import from those packages and make the patcher to break.